gbs-control.ino

```diff
@@ -7415,7 +7472,12 @@ void loop() {
   static unsigned long lastTimeSyncWatcher = millis();
   static unsigned long lastTimeSourceCheck = 500; // 500 to start right away (after setup it will be 2790ms when we get here)
   static unsigned long lastTimeInterruptClear = millis();
-
+  
+  settingsMenu();
```

Comment:
Here it's very obvious how the "settingsMenu" could be confused for a GBSC funtion :)

Reply:
Probably will need OLED rename to show its part of that instead

Reply:
settingsMenuOLED(); then yep :)

Reply:
edited to reflect this change